apps/api/src/routers/admin.py

```diff
@@ -48,11 +48,16 @@ class HackerApplicantSummary(BaseRecord):
     last_name: str
     status: str
     decision: Optional[Decision] = None
-    num_reviewers: int
+    reviewers: list[str] = []
     avg_score: float
     application_data: ApplicationDataSummary
 
 
+class ReviewRequestModel(BaseModel):
```

Comment:
Nitpick: including "Model" in the name feels a bit redundant (like Hungarian notation).